sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -413,13 +424,22 @@ export abstract class ECertFileHandler extends ESDCFileHandler {
         eCertFeedbackResponseRecord.errorCode5,
       ]
         .filter((errorCode) => errorCode)
-        .map((errorCode) => eCertFeedbackErrorCodeMap[errorCode]);
-
+        .map((errorCode) => eCertFeedbackErrorCodeMap[errorCode].id);
+      const blockFundingErrorIds = [
```

Comment:
Calling the variable "id" can be misleading since it will store the "error codes", right?
Can it be renamed to "blockingErrorCodes" which would also demand renaming the parameter in the method `createECertErrorRecord`?

Reply:
Yes